src/main/webapp/main.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Animal shelter</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://www.gstatic.com/firebasejs/ui/6.0.1/firebase-ui-auth.js"></script>
+    <link type="text/css" rel="stylesheet" href="https://www.gstatic.com/firebasejs/ui/6.0.1/firebase-ui-auth.css" />  
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Bowlby+One+SC&display=swap" rel="stylesheet"> 
+    <!-- <script>
+    var config = {
+        apiKey: "AIzaSyAbEVEhhUgDvvtkbu3a-EHPkYee32bz-_c",
+        authDomain: "summer22-sps-24.firebaseapp.com",
+    };
+    firebase.initializeApp(config);
+    </script> -->
+  </head>
+  <body>
+    <button id="logout" onclick="signout()">Log Out</button>
+    <div class = "title-style">
+        <h1>Animal Shelter</h1>
+        <p>A HOME FOR ANIMALS</p>
+    </div>
+    <!-- <script type="text/javascript">
```

Comment:
Are these comments necessary? they also include apikey which is not good practice